src/components/FilterSearch.tsx

```diff
@@ -78,6 +94,11 @@ export interface FilterSearchProps {
   searchOnSelect?: boolean,
   /** A function which is called when a filter is selected. */
   onSelect?: (params: OnSelectParams) => void,
+  /** A function which is called everytime the input element's value changes.
+   * Helpful for removing static filters from the static filter state if a user clears the search input completely.
+   * Replaces the default behavior of calling executeFilterSearch whenever the input changes.
```

Comment:
I think these docs are a little too specific and reference implementation details that the user doesn't necessarily need to know about (e.g. that we call `executeFilterSearch`). Maybe it could be something like this instead: `A function which is called when the input element's value changes. Replaces the default behavior.` The specific example you gave could be added to the HH documentation or a starter repo